Source/Scene/Cesium3DTileset.js

```diff
@@ -309,6 +320,31 @@ define([
         });
     };
 
+    function loadRefiningTiles(refiningTiles) {
+        // Tiles that use replacement refinement and have empty child tiles need to keep track of
+        // descendants with content in order to refine correctly.
+        var stack = [];
```

Comment:
Do you want to generalize the tree traversal, and then pass a function to it here and in `destroy`?